src/peft/helpers.py

```diff
@@ -146,3 +148,20 @@ def check_if_peft_model(model_name_or_path: str) -> bool:
         is_peft_model = False
 
     return is_peft_model
+
+
+@contextmanager
+def set_adapter_scale(model, alpha):
```

Comment:
Let's add a nice docstring to this function. Also, let's update the docs.

Reply:
Added an initial doctring to the function. I think we need to iterate on the docstring and make it more verbose.